playground/infrastructure/ci_helper.py

```diff
@@ -32,7 +35,7 @@ def verify_examples(self, examples: List[Example]):
         2. Group code of examples by their SDK.
         3. Run processing for all examples to verify examples' code.
         """
-        get_statuses(examples)
+        asyncio.run(get_statuses(examples))
```

Comment:
We should await this method(`await get_statuses(examples)`) and run verify_examples() in ci_cd.py with `asyncio.run`

Reply:
Changed.